ci/images/debian10/Dockerfile

```diff
@@ -6,13 +6,13 @@ RUN apt-get update
 RUN apt-get install -y curl gnupg
 
 # Installs node.
-RUN curl -fsSL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+RUN curl -fsSL https://deb.nodesource.com/setup_12.x | bash - \
+  && apt-get install -y nodejs
 
 # Installs yarn.
-RUN curl -fsSL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt-get update && apt-get install -y yarn
+RUN curl -fsSL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+  && apt-get update && apt-get install -y yarn
 
 # Installs VS Code build deps.
```

Comment:
In general I think we should condense these RUN commands, each one creates a separate copy-on-write layer and this can lead to "wasted" space. Not something to do now, but just something to keep in mind for the future :)

When you install packages or run an update, apt creates cache files in `/var/cache/apt` and some other places (it's downloaded packages and the package list), and these consume space in the container image unless you remove them in the same layer (using `apt-get clean`). There's a [magic hook in the Debian docker image](https://github.com/debuerreotype/debuerreotype/blob/d29dd5e030525d9a5d9bd925030d1c11a163380c/scripts/debuerreotype-minimizing-config#L85-L100) that does this automatically, but it's usually better to be explicit (update, install, and clean all in the same step) rather than rely on that magic.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

https://dev.to/cloudx/analyzing-the-docker-layers-with-dive-5e7o